### ISSUE TITLE
Add outside temp

### ISSUE THREAD

**Issue by randyarrasmith** [dieselrabbit/screenlogicpy#13]
First, I like to say how awesome this is.  I have waited a long time for something like this and now I can get pool data in Home Assistant and run automations.  Thank you for making this.

If you are able, it would be great to have outside temperature also included.  The pool sensor is my only outside temperature sensor and I find it more reliable than the reported temperature for my area.  To be able to have that data in available in a script would be very awesome.

**Comment by dieselrabbit** [comment]
This should already be available as the air temperature sensor in both the JSON output of this library, but also in Home Assistant.
Likely it would show up as `sensor.pentair_xx_xx_xx_air_temperature`.

**Comment by jefframsey0** [comment]
> This should already be available as the air temperature sensor in both the JSON output of this library, but also in Home Assistant.
> Likely it would show up as `sensor.pentair_xx_xx_xx_air_temperature`.

Just a FYI. It displays the air temp for mine just fine.

**Comment by randyarrasmith** [comment]
Thank you for the quick reply.
I am slightly embarrassed.  I only did the basic testing on this and didn't see air temp in the default command line output nor in the readme for any 'get' command.  I didn't realize it was in the json data.  Awesome!!! 
I also did not realize that HA 2021.4 is actually using screenlogicpy.  I was going to write my own script to parse the data I want.  Now I don't have to... LOL.   So another big thank you for that!
side note:  my desktop has python 3.7 and screenlogicpy works fine.  My laptop has python 3.9 and it errors with a missing parameter for cli().  So just a heads up when moving forward with 3.9.  I'll leave it up to you if it needs an issue written for it.
Thank you very much for this and also working with HA.
I will close the issue
